R/groups.R

```diff
@@ -180,7 +175,7 @@ clean_group <- function(id, ask = TRUE) {
         }
 
         # Delete child groups
-        child_groups <- rev(get_groups(groups = id))
+        child_groups <- get_groups(groups = id)
```

Comment:
Do you test here? I believe I have to use function `rev` to delete groups.

Reply:
This file got caught up in the merge inadvertently, it's removed now